README.md

```diff
@@ -32,7 +32,7 @@ or you are just interested in a specific part of the project, but not sure where
  to begin, feel free to ask.
 On the other hand if the contribution you wish to make has not been documented 
 in an existing issue, please [create new issue](https://github.com/andela-efalayi/docster/issues/new) before submitting your [pull request](https://help.github.com/articles/about-pull-requests/).
-Please follow the Pull Request decsription convention [here](https://github.com/andela-efalayi/docster/wiki/Pull-Request-Naming-and-Description-Convention)
+Also, kindly follow the Pull Request decsription convention [here](https://github.com/andela-efalayi/docster/wiki/Pull-Request-Naming-and-Description-Convention)
```

Comment:
So, I noticed there are no badges (travis, coveralls and codeclimate) in your readme file. Please integrate them.
Also, I think you should specify your api routes in your readme file.